pages.it/windows/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Mostra informazioni riguardo alla directory corrente e permette di muoversi in un altra directory.
+> Ulteriori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/cd>.
+
+- Mostra il percorso(path) della directory corrente:
+
+`cd`
+
+- Spostati nella directory superiore:
+
+`cd ..`
+
+- Spostati ad una specifica directory sullo stesso disco:
+
+`cd {{percorso\alla\directory}}`
```

Comment:
I am not sure about the path. I will request a review from a native speaker. (Ref: 9031)

Reply:
That should be right and easy to test, though it's a fair doubt. Thanks for the suggestions ^^